package.json

```diff
@@ -15,7 +15,7 @@
     "imports": "babel-node --extensions .ts scripts/count-imports/index.ts",
     "lint": "npm run lint:css && npm run lint:ts",
     "lint:css": "stylelint 'src/Style/**/*.ts'",
-    "lint:ts": "eslint 'src/**/*.{ts,tsx}'",
+    "lint:ts": "eslint '**/*.{ts,tsx}'",
```

Comment:
why we need to lint every file and also do `yarn storybook:build` in validate?

Reply:
This line change to help us lint `stories`.

Add `yarn storybook:build` to make sure the storybook can be deployed successfully. And it can throw the ts error when building now, due to this line https://github.com/glints-dev/glints-aries/pull/272/commits/594ea6aeb222afebae4825bea2632ed5393c7c69#diff-e5e546dd2eb0351f813d63d1b39dbc48R23

Both to ensure that the storybook can be deployed successfully.

Reply:
And I think it's okay to check in each push rather than release.